index.html

```diff
@@ -249,928 +346,829 @@
+          <button onclick="" class="button custom-focus">Buy Tickets</button>
         </div>
-        <div class="tile is-parent">
-          <div class="tile is-child box">
-            <div class="image">
-              <img src="./img/mid_300_33455472_313632022502906_5761920685983137792_n.png">
-            </div>
-            <p class="title">Minsk</p>
-            <p class="subtitle">Fri 27 Nov 2016</p>
-            <a class="button">Buy Tickets</a>
+      </div>
+      <div class="tile is-parent">
+        <div class="tile is-child box">
+          <div class="image">
+            <img src="./img/mid_300_33455472_313632022502906_5761920685983137792_n.png" alt="CSS AND JS in Minsk">
           </div>
+          <h2 class="title">Minsk</h2>
+          <h3 class="subtitle">Fri 27 Nov 2016</h3>
+          <button onclick="" class="button custom-focus">Buy Tickets</button>
         </div>
-        <div class="tile is-parent">
-          <div class="tile is-child box">
-            <div class="image">
-              <img src="./img/access.jpg">
-            </div>
-            <p class="title">London</p>
-            <p class="subtitle">Fri 27 Nov 2016</p>
-            <a class="button">Buy Tickets</a>
+      </div>
+      <div class="tile is-parent">
+        <div class="tile is-child box">
+          <div class="image">
+            <img src="./img/access.jpg" alt="accessibility summit">
           </div>
+          <h2 class="title">London</h2>
+          <h3 class="subtitle">Fri 27 Nov 2016</h3>
+          <button onclick="" class="button custom-focus">Buy Tickets</button>
         </div>
       </div>
     </div>
   </div>
+</section>
 
-  <div class="section">
-    <div class="section-heading has-text-centered">
-      <div class="title is-2">Services</div>
-      <div class="subtitle is-5">What can I do for you?</div>
-    </div>
-    <div class="container">
-      <div class="columns">
-        <div class="column">
-          <a class="box">
-            <div class="content">
-              <div class="title is-5">Front End Web Development</div>
-              Develop Front End using latest standards with HTML5/CSS3 with added funtionality using JavaScript and
-              Vue.js.
-            </div>
-          </a>
-        </div>
-        <div class="column">
-          <a class="box">
-            <div class="content">
-              <div class="title is-5">Back End Web Development</div>
-              Develop Back End application/service using Node.js or ASP .NET and SQL server or Mongo DB databases.
-            </div>
-          </a>
-        </div>
+<section class="section" aria-labelledby="services">
+  <div class="section-heading has-text-centered">
+    <h2 class="title is-2" id="services">Services</h2>
+    <h5 class="subtitle is-5">What can I do for you?</h5>
+  </div>
+  <div class="container">
+    <div class="columns">
+      <div class="column">
+        <a role="button" class="box custom-focus">
```

Comment:
Not focusable by keyboard

Reply:
It's better to use "button" tag instead of the "role" attribute or add tabindex=0

Reply:
i'll use the second variant, if u don't mind